docs/tables/azuread_directory_role.md

```diff
@@ -0,0 +1,41 @@
+# Table: azuread_directory_role
+
+Azure Active Directory (Azure AD) lets you use directory roles to manage access to your cloud-based resources. Azure AD directory roles are also known as administrator roles.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  id,
+  display_name,
+  description
+from
+  azuread_directory_role;
+```
+
+### List directory roles with member ids
```

Comment:
I'd probably remove this example and just put `member_ids` in the `Basic info` example